scaffolder-templates/gitlab-workflows/basic-workflow/docs/GitlabBasicDoc.md

```diff
@@ -0,0 +1,135 @@
+# Creating a Basic Serverless Workflow via a Software Template with Gitlab
+
+## Overview
+This template allows the user to generate a basic workflow project from the user's input.
+The workflow will be generated corresponding to the workflow type (assessment, infrastructure) the user selects.
+
+### basic-workflow as assessment
+An assessment workflow project is generated from the Github Basic Workflow or the Gitlab Basic Workflow templates. This is a simple workflow that evaluates the user's input as string by comparing it to the word `default` to print out the default workflow options or an empty workflow options list.
+Unlike the infrastructure workflow, the output schema of the assessment workflow is crucial to get the workflow options rendered in the UI. In this basic assessment, there is no data output schema validation but the generated workflow provides a sample object for reference which is made of six mandatory fields (currentVersion, upgradeOptions, migrateOptions, newOptions, continuationOptions, otherOptions). See below an example:
+
+```
+"workflowOptions": {
+    "currentVersion": {
+        "id": "DefaultInfrastructureWorkflowTest",
+        "name": "Default infrastructure workflow test"
+    },
+    "upgradeOptions": [],
+    "migrationOptions": [
+        {
+            "id": "MigrationInfrastructureWorkflowTest",
+            "name": "Migration infrastructure workflow test"
+        }
+    ],
+    "newOptions": [],
+    "continuationOptions": [],
+    "otherOptions": []
+}
+```
+
+### basic-workflow as infrastructure
+An infrastructure workflow project is generated from the Github Basic Workflow or the Gitlab Basic Workflow templates. This template prints out the workflow's data input.
+
+## Prerequisites
+
+- An OCP/k8s cluster on which to run RHDH. 
+- An account on [Quay.io](https://quay.io/repository/) 
+- Setting up the Gitlab prerequisites according to the following:
+
+### Prerequisite - Gitlab configuration
```

Comment:
So this is targeted towards gitlab admins; the users using the template must only check that this is already in place?